client/src/launchGdcMatrix.js

```diff
@@ -137,14 +148,11 @@ export async function init(arg, holder, genomes) {
 							chartType: 'matrix',
 							settingsKey: 'maxGenes',
 							callback: async value => {
```

Comment:
i think this max number gene limit should not be defined here as a custom input. it should be coded in matrix and applied to other dataset. can assign to Congyu?

Reply:
@congyu-lu we can discuss how to move this code to matrix.